batch/batch/server/server.py

```diff
@@ -442,14 +448,50 @@ def to_dict(self):
         return result
 
 
+authorized_users = [
+    'ci',
+    'pipeline-test-0-1--hail-is',
+    'google-oauth2|konradk@broadinstitute.org',
+    'google-oauth2|labbott@broadinstitute.org',
+    'google-oauth2|dking@broadinstitute.org',
+    'google-oauth2|akotlar@broadinstitute.org',
+    'google-oauth2|jigold@broadinstitute.org',
+    'google-oauth2|cseed@broadinstitute.org'
+]
+
+
+def authorized_users_only(fun):
+    def wrapped(request, *args, **kwargs):
+        user = request.headers.get('Hail-User', '')
```

Comment:
Where does the cookie / authorization verification come into this?

Reply:
I added JWT decoding, that ensures its a valid cookie issued by us